core/src/main/java/com/linecorp/armeria/client/circuitbreaker/FailFastException.java

```diff
@@ -27,7 +31,11 @@
 
     private final CircuitBreaker circuitBreaker;
 
-    FailFastException(CircuitBreaker circuitBreaker) {
+    /**
+     * Construct an {@link FailFastException} that a request has been failed by {@code circuitBreaker}.
```

Comment:
How about just `Creates a new instance with the specified {@link CircuitBreaker}.`

Reply:
No need for `FastFailException`?

I updated the doc.